scripts/generate_upload_requirements_layer.sh

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
+# This script is used to generate a dependency layer for the lambdas to utilize
+# It's not really for direct usage, `make generate-dependencies` can be used
+# but its part of the `make deploy` 
+
+set -euo pipefail
+shopt -s nullglob;
+
+account_id=$(aws sts get-caller-identity | jq -r ".Account" )
+populated_dss_tf_backend=$(echo ${DSS_TERRAFORM_BACKEND_BUCKET_TEMPLATE} | sed 's/{account_id}/'"${account_id}"'/'  )
```

Comment:
Consider renaming `populated_dss_tf_backend` to `layers_bucket`

Reply:
changed